index.bs

```diff
@@ -577,66 +577,170 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		The [=clipboard=] attribute must return the {{Navigator}}'s [=clipboard=] object.
+		It is used to execute read/write operations from/to the [=system clipboard=].
 		</div>
 
 		</div><!-- dfn-for Navigator -->
 
-	<h3 id="clipboard-interface">Clipboard Interface</h3>
-
+	<h3 id="clipboard-item-interface">ClipboardItem Interface</h3>
 		<pre class="idl" data-highlight="webidl">
-		typedef sequence&lt;ClipboardItem> ClipboardItems;
+			typedef (DOMString or Blob) ClipboardItemDataType;
+			typedef Promise&lt;ClipboardItemDataType&gt; ClipboardItemData;
 
-		[SecureContext, Exposed=Window] interface Clipboard : EventTarget {
-		  Promise&lt;ClipboardItems> read();
-		  Promise&lt;DOMString> readText();
-		  Promise&lt;undefined> write(ClipboardItems data);
-		  Promise&lt;undefined> writeText(DOMString data);
-		};
+			callback ClipboardItemDelayedCallback = ClipboardItemData ();
 
-		typedef (DOMString or Blob) ClipboardItemDataType;
-		typedef Promise&lt;ClipboardItemDataType> ClipboardItemData;
+			[SecureContext, Exposed=Window]
+			interface ClipboardItem {
+				constructor(record&lt;DOMString, ClipboardItemData> items,
+							optional ClipboardItemOptions options = {});
 
-		callback ClipboardItemDelayedCallback = ClipboardItemData ();
+				readonly attribute PresentationStyle presentationStyle;
+				readonly attribute FrozenArray&lt;DOMString> types;
 
-		[Exposed=Window] interface ClipboardItem {
-		  constructor(record&lt;DOMString, ClipboardItemData> items,
-		    optional ClipboardItemOptions options = {});
-		  static ClipboardItem createDelayed(
-		      record&lt;DOMString, ClipboardItemDelayedCallback> items,
-		      optional ClipboardItemOptions options = {});
+				Promise&lt;Blob&gt; getType(DOMString type);
+			};
 
-		  readonly attribute PresentationStyle presentationStyle;
-		  readonly attribute long long lastModified;
-		  readonly attribute boolean delayed;
+			enum PresentationStyle { "unspecified", "inline", "attachment" };
 
-		  readonly attribute FrozenArray&lt;DOMString> types;
+			dictionary ClipboardItemOptions {
+				PresentationStyle presentationStyle = "unspecified";
+			};
+		</pre>
+		<p><dfn>Clipboard Item</dfn></p>
+		A [=clipboard item=] is conceptually data that the user has expressed a desire to make shareable by invoking a "cut" or "copy" command.
+		For example, if a user copies a range of cells from a spreadsheet, it will result in one [=clipboard item=]. If a user copies a set of files from their desktop, that list of files will be a different single [=clipboard item=].
+		Some platforms may support having more than one [=clipboard item=] at a time on the [=Clipboard=], while other platforms replace the previous [=clipboard item=] with the new one. 
+
+		Each [=clipboard item=] can be represented as multiple mime-types. In the example where the user copies a range of cells from a spreadsheet, it may be represented as an image (image/png), an HTML table (text/html), or plain text (text/plain).
```

Comment:
Nit-picking: "Each" -> "a" since this section is about one clipboard item.

Reply:
Done.